<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsvTuonti</class>
 <widget class="QDialog" name="CsvTuonti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV-tuonti</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tuo</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="kirjausRadio">
        <property name="text">
         <string>Kirjauksia</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="oteRadio">
        <property name="text">
         <string>Tiliote tilille</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="TilinvalintaLine" name="tiliEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tuontiTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Otsikko</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tyyppi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tuonti</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Esimerkki</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="muuntoRadio">
     <property name="text">
      <string>Tilinumeroiden muunto</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ohje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/ohje.png</normaloff>:/pic/ohje.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header>db/tilinvalintaline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CsvTuonti</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CsvTuonti</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oteRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>tiliEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kirjausRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>muuntoRadio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
